data/endpoints/fps-initiate-payment-v3.json

```diff
@@ -1,777 +1,816 @@
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "Information supplied by the remitter (in a structured form), to reconcile an entry with item(s) that the payment intends to settle (e.g., a purchase reference number)."
+          },
+          "CounterpartAccountGenericIdentificationScheme": {
+              "type": "object",
+              "properties": {
+                  "code": {
+                      "enum": [
+                          "BBAN",
+                          "CUID",
+                          "UPIC"
+                      ],
+                      "type": "string",
+                      "description": "Name of the identification scheme in coded form. No need to specify if you are providing this information in free-form text (i.e., in the \"proprietary\" field).",
+                      "nullable": true
+                  },
+                  "proprietary": {
+                      "maxLength": 35,
+                      "minLength": 0,
+                      "type": "string",
+                      "description": "Name of the identification scheme in free-form text. No need to specify if you are providing this name in coded form (i.e., in the \"code\" field).",
+                      "nullable": true
+                  }
+              },
+              "additionalProperties": false,
+              "description": "Information about the identification scheme in coded form or free-form text."
+          },
+          "CreateReturnCreditTransferRequest": {
+              "required": [
+                "paymentReturnInstruction"
+              ],
+              "type": "object",
+              "properties": {
+                "paymentReturnInstruction": {
+                  "$ref": "#/components/schemas/PaymentReturnInstruction",
+                  "description": "Information about the payment return."
+                }
+              },
+              "additionalProperties": false,
+              "description": "Information that should be used to create a single credit return."
+          },
+          "HalLink": {
+              "required": [
+                  "href",
+                  "name"
+              ],
+              "type": "object",
+              "properties": {
+                  "name": {
+                      "type": "string",
+                      "description": "The named link type."
+                  },
+                  "href": {
+                      "type": "string",
+                      "description": "URI to be used."
+                  },
+                  "templated": {
+                      "type": "boolean",
+                      "description": "Specifies \"true\" if href is a URI template, i.e., with parameters. Otherwise, this property is absent or set to false."

```

Comment:
Ibid.

Reply:
```suggestion
                      "description": "Specifies \"true\" if href is a URI template, that is, with parameters. Otherwise, this property is absent or set to false."

```